OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -105,10 +115,13 @@ open class OptimoveConfigBuilder: NSObject {
     private var _runtimeInfo: [String: AnyObject]?
     private var _sdkInfo: [String: AnyObject]?
     private var _isRelease: Bool?
+    private var environment: String?
+    private var tenantId: Int?
+    private var brandGroupId: String?
 
-    public convenience init(optimoveCredentials: String?, optimobileCredentials: String?) {
+    public convenience init(optimoveCredentials: String?, optimobileCredentials: String?, preferenceCenterCredentials: String? = nil) {
```

Comment:
zzz, where is `enablePreferenceCenter()` ? PC credentials are deliberately not included in builder constructor arguments because this is a "niche" feature compared to other 2 and didnt want to clutter everyones initialisation with it. Argument list forever growing with an addition of every new feature is i think less neat than calling `enable` on the builder

Well, the reasoning can be dubious or not as usual, but that's what we have in android and slack